specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/examples/OperationsList.json

```diff
@@ -14,6 +14,15 @@
               "operation": "Query resources",
               "description": "Submits a query on resources within specified subscriptions"
             }
+          },
+          {
+            "name": "Microsoft.ResourceGraph/queries/action",
+            "display": {
+              "provider": "Microsoft Resource Graph",
+              "resource": "Queries",
+              "operation": "CRUD operations of saved queries",
```

Comment:
Suggest removing reference to saved word. It's implicit as we are storing as the object. We can add description that it can be used to get the results instead of passing the query at run time.